src/main/java/uk/gov/pay/ledger/transaction/model/CsvTransactionFactory.java

```diff
@@ -99,7 +99,19 @@ public Map<String, Object> toMap(TransactionEntity transactionEntity) {
 
                 result.put(FIELD_GOVUK_PAYMENT_ID, transactionEntity.getExternalId());
                 result.put(FIELD_AMOUNT, penceToCurrency(transactionEntity.getAmount()));
-                result.put(FIELD_TOTAL_AMOUNT, penceToCurrency(totalOrAmount));
+
+                if (transactionEntity.getState() == TransactionState.SUCCESS) {
+                    result.put(FIELD_TOTAL_AMOUNT, penceToCurrency(totalOrAmount));
+                }
+                else {
```

Comment:
We can combine this to just be `else if` to avoid nesting? 

Also, we put `else` on the same line as the closing `}`. We follow whatever rules intelliJ uses to format code, so best to just press `Command + Option + L` before you commit a file.